test/connectors/CustomsFinancialsApiConnectorSpec.scala

```diff
@@ -397,11 +487,29 @@ class CustomsFinancialsApiConnectorSpec extends SpecBase {
     }
   }
 
+  "Populating CashStatementRequests" should {
```

Comment:
These test cases do not belong here. These must be in separate Test class under models.request.CashDailyStatementRequestSpec

hence remove from here